lemur/plugins/lemur_entrust/plugin.py

```diff
@@ -313,10 +313,88 @@ class EntrustSourcePlugin(SourcePlugin):
 
     author = "sirferl"
     author_url = "https://github.com/sirferl/lemur"
+    options = [
```

Comment:
is this needed, and if so, shall we make a better use of it than dummy?

Reply:
Hi @hosseinsh ,
this relates to https://github.com/Netflix/lemur/issues/2105 . Someday I will fix this problem as well...
So sadly , we have to leave it in at the moment.